interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/CcdaServiceDocumentRequestor.php

```diff
@@ -63,7 +63,7 @@ public function socket_get($data)
                     exec($cmd . " > /dev/null &");
                 }
                 sleep(2); // give cpu a rest
-                $result = socket_connect($socket, "127.0.0.1", "6661");
+                $result = socket_connect($socket, "localhost", "6661");
```

Comment:
Why? Did you have issue using the address 127.0.0.1? Generally it's a better idea to use this IP address because localhost is not guaranteed to resolve to 127.0.0.1 and is a loopback and has to be looked up as any other domain/DNS.
The only issue I can think of using 127.0.0.1 is firewall. localhost is not passed through network card and firewall however all firewalls i've seen 127.0.0.1 and 0.0.0.0 are automatic.

QRDA service manager and other places using node is using 127.0.0.1 for socket including the ccda service which listens to 127.0.0.1. Using localhost makes me feel nervous and 127.0.0.1 has always been used since I put node into openemr.

If this proves to be an issue and localhost is found to be a better way to go then we need to refactor everywhere 127.0.0.1 is used.